processing/src/main/java/io/druid/query/filter/ValueGetter.java

```diff
@@ -27,5 +29,7 @@
   // converted to strings. We should also add functions
   // for these and modify ColumnComparisonFilter to handle
   // comparing Long and Float columns to eachother.
+  // Returns null when the underlying Long/Float value is null.
```

Comment:
Not addressed: https://github.com/apache/incubator-druid/pull/5452#discussion_r200339349

Reply:
done.